### ISSUE TITLE
changefeeds.squash_* tests failing

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#4593]
I just enabled the `./test/run changefeeds.squash_*` tests in 08f18cc and they are failing. For example:
```
FAIL changefeeds.squash_true_multi
+ [53, 53, 53]
?   ^   ^   ^


----------------------------------------------------------------------
Ran 20 tests in 94.748s

FAILED (failures=1)
Command '$RETHINKDB/test/changefeeds/squash_true_multi.test' returned non-zero exit status 1

FAIL changefeeds.squash_true_secondary
  File "/Users/larkost/projects/rethinkdb/test/changefeeds/squash_base.py", line 148, in test_insert
    initial[x]["new_val"][self.field])
AssertionError: 493 != 494

----------------------------------------------------------------------
Ran 20 tests in 92.862s

FAILED (failures=2)
Command '$RETHINKDB/test/changefeeds/squash_true_secondary.test' returned non-zero exit status 1
```

Giving to @VeXocide since these are his.

**Comment by VeXocide** [comment]
In CR 3110 by @larkost, turns out the tests weren't taking into account that the initial values are now sorted when running an `.order_by` query.

**Comment by VeXocide** [comment]
Merged into `next` via commit 26b12e47c7e38121ac3d8d2cf1ac8b80e4b30230.
